pmm/pmm2-ami-staging-start.groovy

```diff
@@ -166,6 +196,38 @@ pipeline {
                 archiveArtifacts 'INSTANCE_ID' 
             }
         }
+        stage('Enable Testing Repo') {
+            when {
+                expression { env.ENABLE_TESTING_REPO == "yes" }
```

Comment:
I see double if statement in "Enable Experimental repo",
Shouldn't `expression { env.ENABLE_TESTING_REPO == "yes" }` be `expression { env.ENABLE_TESTING_REPO == "yes" && env.ENABLE_EXPERIMENTAL_REPO == "no" }`?

Reply:
you are right, I basically don't want to check the Experimental Repo flag if Testing Repo has been set to true because that would save us from adding another parameter to calling jobs to set this flag to No every time when we want to enable testing repo, 

from user's point of view, if I ask for enabling testing repo, the job should enable testing repo.